cluster-master.js

```diff
@@ -307,15 +315,23 @@ function forkListener () {
 
     worker.on("disconnect", function () {
       debug("Worker "+id+" disconnect")
-      // give it 1 second to shut down gracefully, or kill
-      disconnectTimer = setTimeout(function () {
-        debug("Worker "+id+", forcefully killing")
-        worker.process.kill("SIGKILL")
-      }, gracefulShutdownDelayMs)
+      startDisconnectTimer(worker, id)
     })
   })
 }
 
+function startDisconnectTimer(worker, id) {
+  clearTimeout(worker.disconnectTimer)
+
+  debug("Worker "+id+" starting disconnect timer")
+
+  // give it 1 second to shut down gracefully, or kill
+  worker.disconnectTimer = setTimeout(function () {
+    debug("Worker "+id+", forcefully killing")
```

Comment:
Should we change the level of this log message to `warn` so we see it in shutdown logs if we hit this timeout?

Reply:
We never see output from anything in this `cluster-master` module because `ojos` explicitly swallows all messages.. see: https://github.com/movableink/ojos/blob/cb4551d4b80a22840eec5b743898fd36031bd7bf/cluster.js#L9